client/Components/Common/Header.jsx

```diff
@@ -29,38 +33,47 @@ class Header extends Component {
 
   render() {
     return (
+        !this.state.authenticated 
+      ?
+        <Redirect to ="/"/>
+      : 
       <div>
-        <nav className="navbar navbar-expand-lg navbar-light bg-light">
-          <NavLink className="navbar-brand" to="/recipe">
-            <img src="images/logo.png" width="270" height="59" alt="logo" />
+        <nav className="navbar navbar-expand-lg navbar-light">
```

Comment:
Why not have this other result of the conditional as a component? It can be returned as a JSX element and import here. This will reduce the number of lines here.

Reply:
It's already a component on it own, so I imported it to wherever I need it